chia/cmds/init.py

```diff
@@ -17,8 +17,11 @@
 )
 @click.option("--testnet", is_flag=True, help="Configure this chia install to connect to the testnet")
 @click.option("--set-passphrase", "-s", is_flag=True, help="Protect your keyring with a passphrase")
+@click.option("--experimental-v2-db", is_flag=True, help="Initialize the blockchain database in experimental v2 format")
```

Comment:
If you have any concerns about users finding and using this option, you can add `hidden=True` to omit it from the CLI help.

Reply:
I'm not really concerned about it.. it does say "experimental" after all